troubleshooting/known_issues/gitea_web_ui_requires_two_logins.md

```diff
@@ -0,0 +1,6 @@
+# Known Issue: Logging into the Gitea web UI requires logging in twice
+
+When using the Gitea web UI, users are redirected to a `keycloak` login.
+The redirect is expected, but logging in on this page should log users into Gitea.
+Instead users are being redirected back to Gitea and have to login again through the Gitea web page using the existing git credentials.
```

Comment:
Where do they get they git credentials? I haven't logged into the Gitea web UI. Do they get the credentials as a part of logging into Keycloak? Do they already have those credentials prior to attempting to log in to Gitea in the first place?

Reply:
I've added a link to the documentation for retrieving the git credentials.  It's separate from keycloak.